logstash-core/lib/logstash/config/mixin.rb

```diff
@@ -382,7 +382,7 @@ def validator_find(key)
       return nil
     end
 
-    def validate_value(value, validator)
+    def validate_value(parent_code_reference, value, validator)
```

Comment:
changing this method will break the JDBC Static Filter Plugin, since it implements its own validation and calls this method. https://github.com/logstash-plugins/logstash-filter-jdbc_static/blob/d21bc5f7bcd137fa482d448b2d20a17e1ff3a636/lib/logstash/filters/jdbc_static.rb